lib/views/pages/home/content/ranking/ranking_card.dart

```diff
@@ -0,0 +1,133 @@
+import "package:flutter/material.dart";
+import "package:proxima/models/ui/ranking/ranking_element_details.dart";
+import "package:proxima/models/ui/user_avatar_details.dart";
+import "package:proxima/views/components/content/publication_header/user_profile_pop_up.dart";
+import "package:proxima/views/components/content/user_avatar/user_avatar.dart";
+
+/// A class that displays a ranking card.
+/// The card displays the user's rank, display name, and centauri points.
+/// The card is clickable and opens a user profile pop-up.
+/// The card's color can be customized.
+class RankingCard extends StatelessWidget {
+  const RankingCard({
+    super.key,
+    required this.rankingElementDetails,
+  });
+
+  final RankingElementDetails rankingElementDetails;
+
+  static const userRankrankTextKey = Key("userRankrankText");
```

Comment:
I usually place `static` `const` at the top of the class so we can clearly see them.

Reply:
Same for other `static` `const` in this PR (please apply everywhere).

Reply:
You are completely right ! It improves readability.

Fixed in: [e156bbc](https://github.com/ProximaEPFL/proxima/pull/322/commits/e156bbc1c67662ccb066c9ab05d4a2a2761f83b6)

Thank you !